examples/emodel_pipeline_local_python/README.rst

```diff
@@ -249,28 +272,32 @@ To facilitate the execution of the different steps of the pipeline on Slurm, we
     ./extract.sh
     ./optimisation.sh
     ./analysis.sh
+    ./export.sh
 
 Make sure to configure the necessary variables within these scripts, including setting the ``OPT_EMODEL`` value as well as configuring their sbatch counterpart by setting the ``#SBATCH`` directives according to your job requirements.
 
 These scripts will also generates logs of the different steps for each run to track its progress and capture any issues that may arise during execution. These log files are stored in the ``./logs`` with a naming convention reflective of the operation and its corresponding job identifier (e.g., ``opt_jobid.log``). In addition to individual log files, each step maintains its own historical record (e.g., ``extract_list.log``, ``opt_list.log`` ``analyse_list.log``) . These files are also situated within the ``./logs`` directory, serving as cumulative logs that document the series of runs pertinent to that particular step. Please ensure to check these logs if you encounter issues during the pipeline execution.
 
+When running the optimisation, the script will create several slurm jobs for different optimisation seeds and a githash associated with the run (keep it preciously!), However, if you lose it, you can retrieve the githash from the opt_list.log file associated with each run. The optimisation script also compiles the mod files, assuming they are in the ``./mechanisms`` directory. Note that BluePyEmodel will delete any existing compiled files folder in the home directory before initiating a new optimisation. This is done to ensure that the mechanisms are compiled again if there are any changes.
```

Comment:
typo: (keep it preciously!), -> (keep it preciously!).
you used a comma instead of a point